### ISSUE TITLE
Rules without selectors don't trigger syntax error in injectGlobal or createGlobalStyle

### ISSUE THREAD

**Issue by brandonkal** [microsoft/typescript-styled-plugin#102]
This will produce invalid CSS:
```jsx
import { injectGlobal, createGlobalStyle } from 'styled-components'

injectGlobal`
  color: red; /* should error here... bad syntax */
  body {
    color: red;
  }
`

createGlobalStyle`
 color: red; /* should error here... bad syntax */
  body{color:green  ;}
`
```
The plugin should understand that certain functions do not nest the template literal contents inside a CSS selector. Global styles in this case should trigger the appropriate invalid syntax errors.

**Comment by odiak** [comment]
I have same problem. In addition, VS Code doesn't format CSS inside their tags.

**Comment by andycmaj** [comment]
I ran into the same problem... styled-plugin doesn't seem to format inside global template string.

so i just use

```
createGlobalStyle(css`

...

`)
```

**Comment by andycmaj** [comment]
yeah formatting doesn't seem to work inside global template strings such as `createGlobalStyle`

so i use this workaround:

const globalCss = css`
...
`;

const GlobalStyles = createGlobalStyle`${globalCss}`;
```

**Comment by mjbvz** [comment]
Closing as this package has been deprecated in favor of the [official styled-components/typescript-styled-plugin fork]( https://github.com/styled-components/typescript-styled-plugin)

After updating to that fork, follow up in the styled-components repo if this is still an issue
